Expecto/Expecto.fs

```diff
@@ -1,1351 +1,1352 @@
-namespace Expecto
```

Comment:
You just added Windows-newlines everywhere. I recommend installing the EditorConfig plugin.